sovtokenfees/sovtokenfees/messages/fields.py

```diff
@@ -5,13 +5,35 @@
 from sovtoken.messages.fields import PublicInputsField, \
     PublicOutputsField
 from sovtokenfees.constants import MAX_FEE_OUTPUTS
+from sovtoken.transactions import TokenTransactions
 
+ALLOWED_FEES_TXNS = [
+    TokenTransactions.XFER_PUBLIC.value,
+    "1", #NYM
+    "100", #ATTRIB
+    "101", #SCHEMA
+    "102", #CRED_DEF
+    "113", #REVOC_REG_DEF
+    "114", #REVOC_REG_ENTRY
+]
+
+# WARNING: This validation is a temporary solution!
+# Set of allowed transactions should be moved to config ledger.
 
 class FeesStructureField(MapField):
     def __init__(self, **kwargs):
         super().__init__(NonEmptyStringField(), NonNegativeNumberField(),
                          **kwargs)
 
+    def _specific_validation(self, val):
+        error = super()._specific_validation(val)
+        if error:
+            return "set_fees -- " + error
+
+        for (k, _) in val.items():
```

Comment:
Can this be optomized into list comprehension? If so will you make these changes

Reply:
After talking to @KitHat, We have decided that this is not necessary.